specification/gedcom.md

```diff
@@ -2831,7 +2831,14 @@ A [registry of component subtags](https://www.iana.org/assignments/language-subt
 
 #### `LANG` (Language) `g7:HEAD-LANG`
 
-The language in which the `Text`-typed payloads of all structures in the document appear, except as superseded by a `g7:LANG`.
+The default language using which to interpert any `Text`-typed payloads of all structures without their own
+`g7:LANG` substructure.  Language information is often used for purposes such as language-specific sorting
```

Comment:
"`g7:LANG` substructure" is too specific; `g7:LANG` can apply to peer structures and their substructures, as e.g. `NAME`.`LANG` applying to `NAME`.`GIVN`. This is why I used "superseded by a `g7:LANG` in the previous text.